### PR TITLE
Fix issues found by linters - enable all tests on all repos - remove suppressions

### PR DESCRIPTION
**Pull request by nhosoi**
- Cleaning up yamllint line-length errors
- Cleaning up ansible-lint errors

### CODE REVIEW

File: roles/pcp/tasks/pmcd.yml [hunk 1]
```diff
@@ -64,9 +64,10 @@
   when: item.user is defined
   with_items: "{{ pcp_accounts }}"
 
+# yamllint disable rule:line-length
 - name: Ensure performance metric collector SASL accounts are configured
   shell: |
-    set -o pipefail
+    set -euo pipefail
     sasldblistusers2 -f "{{ __pcp_pmcd_sasldb_path }}" | grep -q "^{{ item.sasluser }}@"
     [ $? -eq 0 ] && exit 0
```
Comment (richm) [inline roles/pcp/tasks/pmcd.yml:72]: I suspect one of these lines is causing problems with the addition of `-u` or `-e`.  I suggest rewriting it like this:
```yaml
  shell: |
    set -euo pipefail
    if ! sasldblistusers2 -f "{{ __pcp_pmcd_sasldb_path }}" | grep -q "^{{ item.sasluser }}@"; then
      echo "Creating new {{ item.sasluser }} user in {{ __pcp_pmcd_sasldb_path }}"
      echo "{{ item.saslpassword }}" | saslpasswd2 -a pmcd "{{ item.sasluser }}"
      chown root:pcp "{{ __pcp_pmcd_sasldb_path }}"
      chmod 640 "{{ __pcp_pmcd_sasldb_path }}"
    fi
```

Comment (nhosoi) [reply roles/pcp/tasks/pmcd.yml:72]: Thank you, @richm. Updating it.

**Comment by nhosoi** [comment]
[citest bad]

**Comment by nhosoi** [comment]
[citest pending]

**Comment by nhosoi** [comment]
I think the CI failure with rhel-x/ansible-2.9 is covered by bz for RHEL9.

**Review by richm — APPROVED** [review]
lgtm

**Pull request merged**